src/services/dashboards/widgets/_base/base-trend/BaseTrendWidget.vue

```diff
@@ -91,9 +94,15 @@ const state = reactive({
         if (!state.data) return [];
         return state.data.map((d) => d[state.groupBy]);
     }),
-    tableFields: computed(() => [
-        { label: state.groupByLabel, name: state.groupBy },
-    ]),
+    tableData: computed(() => getRefinedDataTableData(state.data, state.groupBy)),
+    tableFields: computed<DataTableField[]>(() => {
```

Comment:
issue: tableFields' type is not DataTableField[], but Field[] which is for WidgetDataTable's fields prop.

Reply:
Oh, I see!